action.yml

```diff
@@ -25,6 +25,9 @@ inputs:
   role-duration-seconds:
     description: "Role duration in seconds (default: 6 hours)"
     required: false
+  role-session-name:
+    description: 'Role Session Name (default: GitHubActions)'
```

Comment:
Nit:
```suggestion
    description: 'Role session name (default: GitHubActions)'
```

Reply:
No problem! I have updated the code based on your suggestions. The test code is ready and looks good, let me know if need more improvements. 😄